ups/ts_dsm.table

```diff
@@ -4,7 +4,6 @@
 #   the "base" package.
 setupRequired(sconsUtils)
 setupRequired(ts_salobj)
-setupRequired(ts_config_eas)
```

Comment:
Please make a separate pull request to remove DSM from ts_config_eas.

Reply:
Thanks for the reminder. I did the work earlier. You should have received a PR for review.

Reply:
Great! I just reviewed it.